.circleci/config.yml

```diff
@@ -190,8 +193,13 @@ workflows:
             tags:
               only: '/^v[0-9].*/'
       - run_automation:
```

Comment:
`run_automation`-s of $CIRCLE_BRANCH and $CIRCLE_TAG need to have same filters as their dependencies (deploy_branch and deploy_tag, respectively), or they will be triggered on all events.